Makefile

```diff
@@ -0,0 +1,9 @@
+SHELL := /bin/bash
+
+.PHONY: py36_test py37_test
+
+py36_test:
+	source travis/scala_classpath_for_python.sh && make -C python py36_test
+
+py37_test:
+	source travis/scala_classpath_for_python.sh && make -C python py37_test
```

Comment:
I suppose the nice separation of concerns would be that `.travis.yaml` & `travis/` (directory) only contain travis-specific conf. Right?

In that sense it seems wrong that `Makefile` uses `scala_classpath_for_python.sh` from under `travis/` when on the other hand `.travis.yaml` uses the `Makefile`. It's not exactly a circular dependency, but mentally yes. Should `scala_classpath_for_python.sh` be moved out of `travis/`?

Reply:
Oh yes, totally. That's a remnant from when the call was in `.travis.yaml`, I'll move it out